packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -321,13 +283,39 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
   public async launchCommunity(payload: InitCommunityPayload) {
     this.logger('Launching community: peers:', payload.peers)
     this.communityState = ServiceState.LAUNCHING
-    // Perhaps we should call this data something else, since we already have a Community type.
-    // It seems like InitCommunityPayload is a mix of various connection metadata.
+
+    // TODO: Move community creation to the backend so that
+    // launchCommunity/createCommunity return a Community object to the
+    // frontend. Also deprecate the COMMUNITY/PSK/OWNER_ORBIT_DB_IDENTITY
+    // IndexDB keys in favor of COMMUNITIES/CURRENT_COMMUNITY_ID/IDENTITIES,
+    // mirroring the frontend state so that we can easily move things from the
+    // frontend to the backend.
```

Comment:
In the next bit of work, I'd like to move the community creation to the backend, where the backend creates and stores the Community object and then returns it to the frontend.